### PR TITLE
[8084] remove igetwild test (main)

### PR DESCRIPTION
**Pull request by trel**
sibling icommands PR at https://github.com/irods/irods_client_icommands/pull/509

### CODE REVIEW

**Review by korydraughn — APPROVED** [review]
Looks good to me.

Let's wait for a second approval before pounding.

**Review by alanking — APPROVED** [review]
Torch it.

**Comment by trel** [comment]
#'d

**Pull request merged**
